back/engines/commercial/id_criipto/app/lib/id_criipto/criipto_omniauth.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module IdCriipto
+  class CriiptoOmniauth < OmniauthMethods::Base
+    include CriiptoVerification
+
+    def profile_to_user_attrs(auth)
+      case config[:identity_source]
+      when DK_MIT_ID
+        custom_field_values = {}
+
+        # Handle birthdate
+        # birthdate is already in YYYY-MM-DD, tak Denmark
+        birthdate = auth.extra.raw_info['birthdate']
+        if (birthday_key = config[:birthday_custom_field_key]) && birthdate
+          custom_field_values[birthday_key] = birthdate
+        end
+
+        # Handle municipality_code
+        municipality_code_str = auth.extra.raw_info.dig('address_details', 'municipality_code')
+        municipality_code_i = municipality_code_str.presence&.to_i
```

Comment:
Reason why I made it a number is because we want to store it as a number field for the smart group queries to be usable? Or do you think an exact match on one or a few of these municipality_codes would be enough, and using text is fine?

Reply:
Interesting point. AFAIK we just need to compare it to the Copenhagen number, 0101. Probably, it's hard to find some ranges among these codes https://en.wikipedia.org/wiki/List_of_municipalities_of_Denmark (though this list has 101 without starting zero).

> Or do you think an exact match on one or a few of these municipality_codes would be enough, and using text is fine?

Yes, it looks so to me. But I'm not super confident about it, so up to you.